notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<ul><li>Please add GitHub and ORCID IDs for all the contributors.</li><li>'codde' misspelled.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>